src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -14,6 +14,7 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException
   { 
-    response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
+  	  // forward request to register.jsp file
```

Comment:
Please align the comment with the start of the expression and capitalize the first letter.